doc/guide/wikiflow.md

```diff
@@ -0,0 +1,370 @@
+  * Import Wikipedia dump (`wikipedia-import`)
+  * Import Wikidata dump (`wikidata-import`)
+  * Construct mapping from Wikipedia to Wikidata (`wikidata-mapping`)
+  * Wikipedia parsing (`wikipedia-parsing`)
+  * Extract and select aliases for entities from Wikidata and Wikipedia (`name-extraction`)
+  * Build name table for searching for entities (`name-table`)
+  * Build phrase table for matching phrases in text to entities (`name-table`)
+  * Merging of Wikipedia categories across languages (`category-merging`)
+  * Fusing information about items to produce final item frame (`item-fusing`)
+  * Build frame store with knowledge base (`knowledge-base`)
+
+After [installing and building SLING](install.md), the `./run.sh` script can be
+used for running the pipeline:
+```
+./run.sh --help
+```
+
+## Download dumps
+
+First, the Wikipedia and Wikidata dumps need to be downloaded:
+```
+./run.sh --download_wikidata --download_wikipedia
+```
+These dumps are large, so it might take a while to download. You can control
+the download with the following flags:
+
+  * `--wikipedia YYYYMMDD`
+    Specifies the version of the [Wikipedia dump](https://dumps.wikimedia.org/enwiki/).
+    If this is not specified, the latest version "sanctioned" by SLING is used.
+    This is usually updated once a quarter.
+  * `--wikidata YYYYMMDD`
+    Specifies the version of the [Wikidata dump](https://dumps.wikimedia.org/wikidatawiki/entities/).
+    If this is not specified, the latest version "sanctioned" by SLING is used.
+    This is usually updated once a quarter.
+  * `--language LANG`
+    Specifies the language for Wikipedia. The default is `en` (English).
+  * `--languages LANG,...`
+    Specifies a list of language for Wikipedia. If `ALL` is specified, the
+    12 "sactioned" languages are downloaded (en,da,sv,no,de,fr,es,it,nl,pt,pl,fi).
+
+This will put the Wikipedia dump into `local/data/corpora/wikipedia` and
+the Wikidata dump into `local/data/corpora/wikipedia`. After the dumps have been
+downloaded, the remaining processing pipeline can be executed in one go:
+```
+./run.sh --build_wiki
+```
+This is equivalent to running each of the step separately:
+```
+./run.sh --import_wikidata
+         --import_wikipedia
+         --parse_wikipedia
+         --merge_categories
+         --fuse_items
+         --build_kb
+         --extract_names
+         --build_nametab
+         --build_phrasetab
+```
+
+## Wikidata import
+
+The Wikidata dump contains _entities_ for all the items in the knowledge base
+in [WikiData JSON format](https://www.mediawiki.org/wiki/Wikibase/DataModel/JSON).
+The `wikidata-import` task reads these and convert them into SLING frame format
+and stores these in a record file set. This also outputs the [schema](https://www.mediawiki.org/wiki/Wikibase/DataModel)
+for the Wikidata properties in SLING frame schema format. After this, the
+`wiki-mapping` task produces a frame store that maps from Wikipedia article ids
```

Comment:
shouldn't it be wikidata-mapping?

Reply:
Seems like my naming is not consistent. I think 'wikipedia-mapping' is the right name. I have changed this throughout the document.